src/test/resources/test_data/insert-schedule-and-offence-data.sql

```diff
@@ -14,24 +14,6 @@ VALUES ('XX99002',
         'CJS Title 2Fail to give to an authorised person information / assistance / provide facilities that person may require',
         '2015-03-13', NULL, '2020-06-17 15:31:26.000', '2022-04-07 16:05:58.178', '2022-04-07 16:05:58.178');
 
-INSERT INTO schedule (act, code, url)
```

Comment:
schedules are now created through ref data - so this becomes redundant for test data set up